From d9755c545ae74a052e6e04d3dfc5b2aa256483ff Mon Sep 17 00:00:00 2001
From: me-cafebabe <me_cafebabe@hotmail.com>
Date: Sat, 20 Aug 2022 15:31:29 +0000
Subject: [PATCH] Remove rules that hits neverallows

I don't use AOSP recovery, so I'm lazy to import these commits in system/sepolicy
---
 common/private/recovery.te | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/common/private/recovery.te b/common/private/recovery.te
index 2b6f7fa..2fea7c1 100644
--- a/common/private/recovery.te
+++ b/common/private/recovery.te
@@ -6,13 +6,13 @@ permissive recovery;
 # Volume manager
 allow recovery block_device:dir create_dir_perms;
 allow recovery block_device:blk_file { create unlink rw_file_perms };
-allow recovery self:capability { mknod fsetid };
+#allow recovery self:capability { mknod fsetid };
 allow recovery proc_filesystems:file r_file_perms;
 allow recovery self:netlink_kobject_uevent_socket create_socket_perms_no_ioctl;
 allow recovery sysfs:file w_file_perms; # writing to /sys/*/uevent during coldboot.
 allow recovery tmpfs:file link;
 allow recovery rootfs:dir w_dir_perms;
-allow recovery rootfs:file { create_file_perms link };
+#allow recovery rootfs:file { create_file_perms link };
 allow recovery media_rw_data_file:dir r_dir_perms;
 
 # Read fbe encryption info
-- 
2.30.2

